kombu/transport/redis_cluster.py

```diff
@@ -384,7 +384,11 @@ def _brpop_start(self, timeout):
                             logger.warning('Error while sending ASKING', extra={"e": e, "key": conn.key})
                             continue
 
-                    conn.client.connection.send_command('BRPOP', key, timeout)
+                    try:
+                        conn.client.connection.send_command('BRPOP', key, timeout)
+                    except:
+                        logger.exception('Error while sending BRPOP', extra={"key": conn.key})
+                        self.connection.cycle._unregister(self, self.client, conn, cmd)
```

Comment:
The cmd variable is not seem to exist in here.
Could you please check it?

Reply:
Thanks!